gpkit/geometric_program.py

```diff
@@ -43,7 +43,8 @@ def __init__(self, cost, constraints, verbosity=1):
         self.cost = cost
         self.constraints = constraints
         self.posynomials = [cost] + list(constraints)
-        NomialData.__init__(self, signomials=self.posynomials, simplify=False)
+        super(GeometricProgram, self).__init__(signomials=self.posynomials,
```

Comment:
Yeah, they're identical. I guess I don't really care that much. I just feel like every time I see this I'm going to wonder "what the heck is it doing?"